packages/mobile/src/utils/linking.ts

```diff
@@ -13,8 +15,12 @@ export function navigateToWalletStorePage() {
   }
 }
 
-export function isDeepLink(url: string): boolean {
-  return url.startsWith('celo:')
+export function isDeepLink(deeplink: string): boolean {
+  return deeplink.startsWith('celo:')
+}
+
+export function isPlaidRedirectDeepLink(deepLink: string): boolean {
+  return decodeURIComponent(deepLink).startsWith(PLAID_UNIVERSAL_LINK_PREFIX)
```

Comment:
Is `decodeURIComponent` necessary?

Reply:
So the reason why using a universal link instead of `celo://` is Plaid requires us to use universal link, See the react native (iOS) section here https://plaid.com/docs/link/oauth/#create-and-register-a-redirect-uri

I'm actually not sure if the decodeURIComponent is necessary, I'll try to test e2e to see if it's something i need to fix.,